### PR TITLE
feat: changed and updated react structure

### PR DESCRIPTION
**Pull request by yuriimartynenko**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,30 +2,12 @@
...
     "test": "react-scripts test --coverage",
-    "eject": "react-scripts eject"
-  },
-  "eslintConfig": {
-    "extends": [
-      "react-app",
-      "react-app/jest"
-    ]
+    "eject": "react-scripts eject",
+    "lint": "eslint src --ext .ts,.tsx"
```
Comment (serhiykrupka) [inline package.json]: change back `start` script to `serve -s build -p 8080` it's needed for build & deploying app with Paketo buildpacks

File: nginx.conf [hunk 2]
```diff
@@ -0,0 +1,50 @@
+server {
```
Comment (serhiykrupka) [inline nginx.conf]: do we need this config?

**Review by serhiykrupka — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (yuriimartynenko) [reply package.json]: done

File: nginx.conf (hunk 2 above)
Comment (yuriimartynenko) [reply nginx.conf]: No. 
Deleted

**Review by andriy-shvydchenko — APPROVED** [review]
/approve

**Pull request merged**